docs/development.md

```diff
@@ -0,0 +1,15 @@
+# Running the SHIELD Dashboard locally
+
+1. `echo DEBUG=True >> .env`
+2. `virtualenv env`
+3. `source env/bin/activate`
+4. `pip install -r requirements.txt`
+5. `npm install`
+6. `gulp build`
+7. Set up the database
+    1. Start a local postgresql server
+    2. Create a new user with name *shielddash* and password *shielddash*
+    3. Create a new database named *shielddash* that the *shielddash* user has
+       read/write access to
```

Comment:
I think it'd be nice to drop the postgres commands in here for this. I could update it after this is merged.